src/app/services/card/card.service.ts

```diff
@@ -41,18 +41,18 @@ export class CardService {
     return this.afs.collection<Card>('static_cards', ref => ref.where('holders', 'array-contains', holderId)).valueChanges({idField:'id'});
   }
 
-  addStaticCard(card: Card): Promise<void> {
```

Comment:
What's difference? (What does "void" mean and do?)

Reply:
All `void` means is that the Promise will resolve with either no data (`null`) or reject with an error. I changed the underlying method used. It originally used a sequence which used AngularFire to pull the `card` collection to then create a new blank card document and set said document to the given input object. Now it uses AngularFire to pull the `card` collection to directly add the input object. Check out the AngularFire documentation and source code to see the differences in return type.

https://github.com/angular/angularfire/blob/master/docs/firestore/documents.md
https://github.com/angular/angularfire/blob/master/docs/firestore/collections.md